### PR TITLE
Add StableHLO Batchnorm operation support to builder ecosystem

### PR DESCRIPTION
**Pull request by velonica0**
### Ticket
Link to Github Issue #4886 

### Problem description
Add Stablehlo Batchnorm ops

### What's changed

1. StableHLO has three operators for BatchNorm. Following the addition of `stablehlo.batch_norm_grad` and `stablehlo.batch_norm_training` in #6270, this PR adds `stablehlo.batch_norm_inference`.
2. Tests have been added for all three BatchNorm operators.

### Checklist
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

**Review by jgrimTT — APPROVED** [review]
lgtm!
